GameBoard/Areas/Identity/Pages/Account/Register.cshtml.cs

```diff
@@ -4,32 +4,41 @@
 using System.Transactions;
 using GameBoard.Areas.Identity.Pages.Account.Notifications;
 using GameBoard.DataLayer.Entities;
+using GameBoard.LogicLayer.Configurations;
+using GameBoard.LogicLayer.Groups;
 using GameBoard.LogicLayer.Notifications;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 
 namespace GameBoard.Areas.Identity.Pages.Account
 {
     [AllowAnonymous]
     public class RegisterModel : PageModel
     {
+        private readonly IGroupsService _groupsService;
         private readonly ILogger<RegisterModel> _logger;
         private readonly INotificationService _notificationService;
         private readonly UserManager<ApplicationUser> _userManager;
+        private GroupsConfiguration GroupsOptions { get; }
```

Comment:
There's no point in a `GroupsOptions` default get-only property. Change this to a `private readonly` field.

Reply:
Marking as done.